packages/sheet/src/sheet.ts

```diff
@@ -65,32 +68,28 @@ export class Sheet {
   }
 
   private _addBaseRule(className: string, css: string) {
-    css = css.replace(cssPropertyRegex, "");
-    this.base.push(`.${className}{${css}}`);
+    const minifiedCss = removeSpacesAroundCssPropertyValues(css);
+    this.base.push(`.${className}{${minifiedCss}}`);
   }
 
   private _addMediaRule(
     className: string,
     css: string,
     breakpoint: string
   ): void {
-    css = css.replace(cssPropertyRegex, "");
-    const mediaCss =
-      `@media (min-width: ${breakpoint}) { .${className} { ${css} } }`.replace(
-        removeSpacesExceptInPropertiesRegex,
-        ""
-      );
+    const minifiedCss = removeSpacesAroundCssPropertyValues(css);
+    const mediaCss = removeSpacesExceptInProperties(
+      `@media (min-width: ${breakpoint}) { .${className} { ${minifiedCss} } }`
+    );
     this.responsive.push(mediaCss);
   }
 
   private _addPseudoRule(
     className: string,
     pseudo: SystemStyle["pseudo"][number]
   ) {
-    const css = pseudo.base.replace(cssPropertyRegex, "");
-    const pseudoCss = `.${className}${pseudo.key} { ${css} }`.replace(
-      removeSpacesExceptInPropertiesRegex,
-      ""
+    const pseudoCss = removeSpacesExceptInProperties(
+      `.${className}${pseudo.key} { ${pseudo.base} }`
```

Comment:
I noticed unintended behavior change. I'll fix here.

Reply:
`removeSpacesExceptInProperties` contains all of `removeSpacesAroundCssPropertyValues` regexp, so this diff causes no behavior change actually.

Reply:
Anyway I added commit c34ace78b0dc028299f022c6d9c5eb41d10d377c

Reply:
@kotarella1110 Could you review change c34ace78b0dc028299f022c6d9c5eb41d10d377c?
I added some refactoring, too.

Reply:
thanks! will review 👍